### PR TITLE
adding a var to ansible and reads with simple deployment

### PR DESCRIPTION
**Pull request by robcxyz**

### CODE REVIEW

File: terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,3 @@
+//region           = "us-east-1"  # See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html
+//private_key_path = "/path/to/your/key"
+//public_key_path  = "/path/to/your/key.pub"
```
Comment (shinyfoil) [inline terraform.tfvars]: Did you mean to include this file/you're referencing AWS docs?

**Review by shinyfoil — APPROVED** [review]
Fine to merge. Just that one question about the tfvars file.

File: terraform.tfvars (hunk 1 above)
Comment (robcxyz) [reply terraform.tfvars]: Ahh good catch.  Can you update that as  you are the alibaba guy. @shinyfoil

Comment (shinyfoil) [reply terraform.tfvars]: Done. Closed.

**Pull request merged**